backend/backend/views.py

```diff
@@ -72,6 +64,17 @@ def get(self, request, *args, **kwargs):
 
         return ApiResponse({"tokens": [_bearer_token_dict(token, include_payload=False) for token in tokens]})
 
+    def delete(self, request, *args, **kwargs):
+        if BearerToken.objects.filter(token_id=request.GET.get("id")):
+            token = BearerToken.objects.get(token_id=request.GET.get("id"))
+            if request.user == token.user:
+                token.delete()
+                return HttpResponse("Authorized, token removed", status=200)
+            else:
+                return HttpResponse("Unauthorized, token was NOT removed", status=401)
+        else:
+            return HttpResponse("Token not found", status=404)
```

Comment:
In this block of code, you calling the DB two times to get the same information (`filter` then `get`).

You could either work with a `tokens = BearerToken.objects.filter()` (and then use the first element) or use directly `BearerToken.objects.get()` and leverage the exception management system. In this case, you would have [2 exception types](https://docs.djangoproject.com/en/4.2/topics/db/queries/#retrieving-a-single-object-with-get) to take in account.

In this case, the second option (leveraging Exception) seems cleaner.